main.py

```diff
@@ -6,16 +6,26 @@
 import os
 import random
 from dotenv import load_dotenv
+import requests
+from requests.auth import HTTPBasicAuth
 
 load_dotenv()
 Token = os.environ.get("BOTTOKEN")
 
+rpc_user = os.environ.get("rpc_user")
+rpc_password = os.environ.get("rpc_password")
+rpc_port = os.environ.get("rpc_port")
+rpc_ip = os.environ.get("rpc_ip")
+from_address = os.environ.get("from_address")
+eurl = os.environ.get("explorer_url")
+ticker = os.environ.get("ticker")
+
 intents = discord.Intents.all()
 bot = commands.Bot(command_prefix='$>', intents=intents)
 
 pattern = re.compile(r"^(C[a-zA-Z0-9]{33})$")
```

Comment:
We should update this to maybe an env variable so others can use it with different wallet addresses.  Maybe add in a link for the readme on how to generate REGEX for their particular wallet.